### ISSUE TITLE
String literals throws errors

### ISSUE THREAD

**Issue by fxck** [angular/vscode-ng-language-service#16]
```typescript
class FooCmp {
  something: 'foo' | 'bar';
}
```

```html
<div *ngIf="something === 'foo'"></div>
```

throws 

> 'Expected the operants to be of similar type or any'

**Comment by chuckjaz** [comment]
Tracking as https://github.com/angular/angular/issues/13403

**Comment by chuckjaz** [comment]
Closed in 0.0.11
